css/styles.css

```diff
@@ -0,0 +1,53 @@
+header {
+  background-color: #4c2676;
+  color: #fff;
+}
+
+h1 {
+  display: flex;
+  margin: 0;
+  padding: 2.5%;
+  width: 100vw;
+}
+
+body {
+  background-color: #eee;
+}
+
+#image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  max-width: 100%;
```

Comment:
This should be `width` instead of `max-width`. We always want this to take up 100% of the it's parent